src/jumping_sumo.rs

```diff
@@ -51,7 +51,7 @@ pub enum PilotingID {
 
 #[derive(Default, Debug, PartialEq, Clone, Copy)]
 pub struct PilotState {
-    pub flag: u8,
+    pub flag: bool,
```

Comment:
`bool` is `u8` as well. Better for the API to use `bool`, instead of `u8`, IF the API doesn't support any other values.